src/CGB_Registers.md

```diff
@@ -191,24 +191,7 @@ TODO: confirm whether interrupts can occur (just the joypad one?) during the pau
 ### FF56 - RP - CGB Mode Only - Infrared Communications Port
```

Comment:
Since other CGB-only regs are mentioned in other pages only (such as palettes), this section should be done away with entirely.

Reply:
Might be good to open an issue for this then and do away with this section entirely - except there *are* some things which I'm not sure belong elsewhere (WRAM banking, speed switching).